.github/actions/git-diff-on-components/README.md

```
@@ -34,8 +36,10 @@ in that way `steps.changed_files.outputs.all` will be converted in array of stri
 ```
 
 ## Build
+
 You need to push all files generated in `dist` folder once you are finished with the build to test the new version of the github action in case you want to make modifications.
+
 ```
 $ cd .github/actions/git-diff-on-components/
-$ npm i && npm run build
-```
\ No newline at end of file
+$ pnpm i && pnpm run build
```

Markdownlint (0.35.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)